ffv1.md

```diff
@@ -429,24 +429,60 @@ Early experimental versions of FFV1 used the CABAC Arithmetic coder from H.264 a
 
 #### Range Binary Values
 
-To encode binary digits efficiently a Range coder is used.
+To encode binary digits efficiently a Range coder is used. A Range coder encodes a series of symbols by using a probability estimation for each symbol (expressed in Quantization Table Sets for an FFV1 encoding) and a range of values that can be divided into sub-ranges proportional to the probability of each symbol encoded. The range is sub-divided until an sub-range can be used to decode the symbol.
```

Comment:
I am not an English speaker, but `until an sub-range` or `until a sub-range`?

Reply:
resolved, thx